CHANGELOG.md

```diff
@@ -17,11 +17,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   from `ContainerFuture` to `CivisFuture` (#426)
 - Updated the docstrings for `file_to_civis` (for `buf` and `expires_at`),
   `dataframe_to_file` (for `expires_at`), and `json_to_file` (for `expires_at`). (#427)
-- Ability to use joblib 1.1.x (#423)
+- Ability to use joblib 1.1.x (#429)
 
 ### Fixed
+- Clarified the `table_columns` parameter in `civis.io.*` functions (#434)
```

Comment:
@gkcivis I didn't remember to ask you to add a changelog entry at #434. I'm adding it back while I'm at it here.

Reply:
thanks!